Xamarin.PropertyEditing.Windows/Xamarin.PropertyEditing.Windows.csproj

```diff
@@ -30,6 +30,11 @@
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.VisualStudioEng.MicroBuild.Core">

```

Comment:
I agree that you can bring these in via `Directory.Build.targets`, whether in this repo or in the repo that submodule this.